%YAML:1.0
---
calibrationTimeChar: "Mon Oct 21 17:22:49 2024\n"
calibrationTime_t: 1729502569
imageWidth: 2240
imageHeight: 1740
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1668873571291749e+03, 0., 1.1344449581056385e+03, 0.,
       1.1644698310764004e+03, 9.0731299575786511e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -9.2406675476366607e-02, 7.9335134036674887e-02,
       3.0640043157793541e-04, -2.3877200465868785e-04,
       -1.8241525739278692e-02 ]
newCameraMatrixAlpha1: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1300264489496728e+03, 0., 1.1342327993688252e+03, 0.,
       1.1261717155633628e+03, 9.0764461374790631e+02, 0., 0., 1. ]
newDistCoeffsAlpha1: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 1.4351213747445615e-04, -5.4913208113576103e-05,
       3.6642105323025650e-05, 2.9868357426143880e-04,
       -6.0606490332406636e-06 ]
newCameraMatrixAlpha0: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1432821363592980e+03, 0., 1.1338586042615484e+03, 0.,
       1.1411399949256322e+03, 9.0705164051346082e+02, 0., 0., 1. ]
newDistCoeffsAlpha0: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 1.0925176327374258e-04, 2.5312414813458147e-05,
       4.0471226825004538e-05, 2.9756631068526373e-04,
       -6.8861796308024013e-05 ]